README.md

```diff
@@ -23,37 +25,87 @@ Primary features:
 To see RedisGraph in action, visit [Demos](https://github.com/RedisGraph/RedisGraph/tree/master/demo).
 To read the docs, visit [redis.io](https://redis.io/docs/stack/graph/).
 
-## Quickstart
+RedisGraph is part of [Redis Stack](https://github.com/redis-stack).
 
-1. [Trying RedisGraph](#trying-redisgraph)
-2. [Docker](#docker)
-3. [Build](#building)
-4. [Start](#loading-redisgraph-into-redis)
-5. [Use from any client](#using-redisgraph)
+## Setup
 
-## Trying RedisGraph
+You can either get RedisGraph setup in a Docker container or on your own machine.
 
-To try RedisGraph, either use the RedisGraph Docker image, or [create a free Redis Cloud Essentials account](https://redislabs.com/try-free/) to get a RedisGraph instance in the cloud.
+### Docker
+To quickly try out RedisGraph, launch an instance using docker:
+```sh
+docker run -p 6379:6379 -it --rm redis/redis-stack-server:latest
+```
 
-## Docker
+### Build it yourself
 
-To quickly tryout RedisGraph, launch an instance using docker:
+You can also build RedisGraph on your own machine. Major Linux distributions as well as macOS are supported.
 
-```sh
-docker run -p 6379:6379 -it --rm redis/redis-stack-server:latest
+First step is to have Redis installed, of course. The following, for example, builds Redis on a clean Ubuntu docker image (`docker pull ubuntu`):
+
+```
+mkdir ~/Redis
+cd ~/Redis
+apt-get update -y && apt-get upgrade -y
+apt-get install -y wget make pkg-config build-essential
+wget https://download.redis.io/redis-stable.tar.gz
+tar -xzvf redis-stable.tar.gz
+cd redis-stable
+make distclean
+make
+make install
 ```
```

Comment:
1. To build RedisGraph we don't need Redis.
2. `apt-get` isn't available on all Linux distros

I wouldn't mention this step.

Reply:
Of course people who already have Redis can skip this part, but the assumption here is that many people don't have Redis yet, and we want to instruct them how to start from nothing and reach the point were they have RedisGraph running.

Re `apt-get` - true, but as I stated - this is an example were you start with a clean Ubuntu docker image.

And again - we have similar build instructions for RTS and RB now. There were many issues like #2982 across all modules - and I believe it is now solved for RTS and RB (anyway - I'll keep syncing all 3 repos on new issues).

Reply:
I also have few notes about this. Sent a DM.